scripts/setup.ts

```diff
@@ -27,11 +27,13 @@ has to point to the dev version you want to promote, for example 2.1.0-dev.123`)
     await run('.', `git config --global user.name "prisma-bot"`)
   }
   if (process.env.RELEASE_PROMOTE_DEV) {
-    const versions = await getVersionHashes(process.env.RELEASE_PROMOTE_DEV)
-    // TODO: disable the dry run here
+    // Checkout exactly the commit of the `dev` version that is being promoted/released
+    
+    const prismaCommit = await getPrismaCommitFromPackageJsonViaUnpkg(process.env.RELEASE_PROMOTE_DEV)
+    // TODO: disable the dry run here // TODO 2: What does this mean?
```

Comment:
😅

Reply:
Honestly, do you know?

Reply:
I think it's about this line
```
await run(`.`, `git checkout ${prismaCommit}`, true)
```

Where true here means dryRun = true

Reply:
Which means this is never executed? I don't like this